internal/boxcli/create.go

```diff
@@ -71,7 +72,7 @@ func runCreateCmd(
 		return err
 	}
 
-	ux.Fsuccess(
+	ux.FSuccess(
```

Comment:
🤔 isn't the golang naming convention better followed by the previous code? For example, consider `fmt.Fprintf` and `fmt.Sprintf` and so on...

Reply:
> 🤔 isn't the golang naming convention better followed by the previous code? For example, consider `fmt.Fprintf` and `fmt.Sprintf` and so on...

Oh, that makes sense.